### PR TITLE
Use Stdext 2.0.0

### PR DESCRIPTION
**Pull request by jonludlam**
- Remove some autogenerated stuff too. 
- Bump version to 1.0.0.
- Add opam file

### CODE REVIEW

**Comment by johnelse** [comment]
Yep :shipit:

File: lib/inventory.ml [hunk 1]
```diff
@@ -13,6 +13,7 @@
  *)
 (* Code to parse the XenSource inventory file *)
 
+open Stdext
```
Comment (lindig) [inline lib/inventory.ml]: Minor quibble: I'm not a fan of many open modules.

**Comment by lindig** [comment]
Looks good to me.

File: lib/inventory.ml (hunk 1 above)
Comment (jonludlam) [inline lib/inventory.ml]: Generally I agree, but this only brings relatively unusually names modules into scope, no vars or types.

**Pull request merged**